packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -1,11 +1,31 @@
 <script lang="ts" setup>
+import { useDensity } from "../../composables/density";
 import type { OnyxTableProps } from "./types";
 
-defineProps<OnyxTableProps>();
+const props = withDefaults(defineProps<OnyxTableProps>(), {
+  striped: false,
+  grid: false,
+});
+
+defineSlots<{
+  /**
+   * Table content (columns + rows).
```

Comment:
should we recommend here to use th/tr/td elements?
if something else is passed, the whole style would not work after all.